src/app/pages/my-openstax/store/sfapi.js

```diff
@@ -0,0 +1,49 @@
+const server = 'https://dev.salesforce.openstax.org'; // 'http://localhost:4004'
+//  /api/v1/opportunities/0060v000008WUp1AAG
```

Comment:
just curious, is there a local mock server for salesforce?  if so, that might be useful for tutor